TestModels/RecursiveShape/Makefile

```diff
@@ -0,0 +1,37 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+CORES=2
+
+#TODO: Think about extern processing in new test models
+ENABLE_EXTERN_PROCESSING=1
```

Comment:
We still need sed processing because `make polymorph_dafny` will generate `module {:extern "simple.recursiveshape.internaldafny.types" } SimpleRecursiveShapeTypes`. Thats why there is a TODO.